README.md

```diff
@@ -28,6 +28,7 @@ docker run \
 --env APOLLO_GRAPH_REF="your graph here" \
 --env APOLLO_KEY="your key here" \
 --env MCP_ENABLE=1 \
+--env MCP_UPLINK=1 \
```

Comment:
I confirmed that this is necessary to be a minimum working command.

![Shot 2025-06-25 at 16 04 14](https://github.com/user-attachments/assets/8b28ca2c-d705-4c5a-8514-61dfdaa12e1c)